src/codewhisperer/service/transformByQHandler.ts

```diff
@@ -84,13 +84,32 @@ export async function getOpenProjects() {
  */
 export async function validateProjectSelection(project: vscode.QuickPickItem) {
     const projectPath = project.description
+    const buildSystem = await checkBuildSystem(projectPath!)
+    if (buildSystem !== BuildSystem.Maven) {
+        vscode.window.showErrorMessage(CodeWhispererConstants.noPomXmlFoundMessage)
+        if (buildSystem === BuildSystem.Gradle) {
```

Comment:
For this case the code below will emit `codeTransform_isDoubleClickedToTriggerInvalidProject` twice. Is that intentional? You will have 2x the number of `result=failed` results when buildsystem=gradle...

Reply:
Good point, fixed